core/gethwrappers/ocr2vrf/go_generate.go

```diff
@@ -7,3 +7,4 @@ package gethwrappers
 //go:generate go run ../generation/generate/wrap.go ../../../contracts/solc/v0.8.15/VRFCoordinator.abi ../../../contracts/solc/v0.8.15/VRFCoordinator.bin VRFCoordinator vrf_coordinator
 //go:generate go run ../generation/generate/wrap.go ../../../contracts/solc/v0.8.15/VRFBeacon.abi ../../../contracts/solc/v0.8.15/VRFBeacon.bin VRFBeacon vrf_beacon
 //go:generate go run ../generation/generate/wrap.go ../../../contracts/solc/v0.8.15/BeaconVRFConsumer.abi ../../../contracts/solc/v0.8.15/BeaconVRFConsumer.bin BeaconVRFConsumer vrf_beacon_consumer
+//go:generate go run ../generation/generate/wrap.go ../../../contracts/solc/v0.8.15/LoadTestBeaconVRFConsumer.abi ../../../contracts/solc/v0.8.15/LoadTestBeaconVRFConsumer.bin LoadTestBeaconVRFConsumer vrf_beacon_consumer_batch
```

Comment:
Any reason this is called `vrf_beacon_consumer_batch` vs. `load_test_beacon_consumer`?

Reply:
I can change it to this. I felt like putting load_test in front was a little non-standard for the way core repo files were named, but it may be less confusing to another dev as such.

Reply:
Ah I guess our other load testing contracts are named `VRFLoadTest...` rather than `LoadTestVRF...`. Personally I think it's okay either way.